code/game/objects/structures/grille.dm

```diff
@@ -21,9 +21,11 @@
 	var/shockcooldown = 0
 	var/my_shockcooldown = 2 SECONDS
 
-/obj/structure/grille/detailed_examine()
-	return "A powered and knotted wire underneath this will cause the grille to shock anyone not wearing insulated gloves.<br>\
-			Wirecutters will turn the grille into metal rods instantly. Grilles are made with metal rods."
+/obj/structure/grille/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>A powered wire underneath this will cause the grille to shock anyone who touches the grill. An electric shock may leap forth if the grill is damaged.</span>"
```

Comment:
```suggestion
	. += "<span class='notice'>A powered wire underneath this will cause the grille to shock anyone who touches the grill. An electric shock may leap forth if the grill is damaged.<br>\
	Use <b>wirecutters</b> to deconstruct this item.</span>"
```